.github/workflows/test.yaml

```
@@ -0,0 +1,17 @@
+on:
+  push:
+    branches:
+      - feat/docker-compose
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)